README.md

```diff
@@ -125,43 +107,57 @@ const hooks = {
 }
 ```
 
-### `store.js`
+### `browser.js`
+
+a function to generate your dogstack client app, which takes as an arg an object of the following properties:
```

Comment:
i think we should include an example `browser.js` importing from the various files described below. i was kinda confused why each the files were described when it's just to use in `browser.js`.

Reply:
👍